ql/src/java/org/apache/hadoop/hive/ql/exec/MoveTask.java

```diff
@@ -774,6 +780,100 @@ private void checkFileFormats(Hive db, LoadTableDesc tbd, Table table)
     }
   }
 
+  class LocalTableLock  implements Closeable{
```

Comment:
Favor static member classes over non static

Reply:
its non-static because it needs access to the txnmanager.